src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,21 @@
 package racingcar;
 
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.Random;
+
 public class Application {
-    public static void main(String[] args) {
-        // TODO: 프로그램 구현
+    public static <string, try1> void main(String[] args) {
+        Scanner in = null;
+        System.out.println("자동차 이름을 적으시오 : ");
+        string cars = (string) in.next();
```

Comment:
자바에서 String은 소문자로 적으시면 안돼요! 대소문자 구분이 확실해야 한답니다
```suggestion
        String cars = (string) in.next();
```

Reply:
+ (string)으로 캐스팅을 진행하신 이유가 있을까요?